### ISSUE TITLE
Adding extra/buntype package

### ISSUE THREAD

**Issue by vmihailenco** [uptrace/bun#112]
@d-fal sorry for the late reaction, but what do you think about moving github.com/pgcontrib/bigint to github.com/uptrace/bun/buntype? So `bigint.Bigint` becomes `buntype.Bigint`.

**Comment by d-fal** [comment]
Sounds good, will go-pg users be invited to use buntype?

**Comment by vmihailenco** [comment]
They are definitely welcome (just like everyone else). Is there anything specific we need to do to better accommodate them?

**Comment by d-fal** [comment]
Since the ultimate fate of go-pg is to be replaced by bun, I think this is a good start to let its users know if they are looking for new features, there is a **bun** out there.

**Comment by vmihailenco** [comment]
Yes, I will add big fat banner to go-pg once we release Bun v1.0 which should happen this month. The only blocking issue is https://github.com/uptrace/bun/issues/59 and docs polishing.

**Comment by d-fal** [comment]
I was testing the package with bigint and I found that when I want to load fixtures from yaml file, I got unmarshall error. Should we equip the extra types with marshall/unmarshallers as well?

**Comment by vmihailenco** [comment]
@d-fal yes, probably, that is main advantage of having these types in the main repo, But perhaps we need a scanner func instead - I am not sure. Perhaps I will know more after https://github.com/uptrace/bun/issues/59 is closed.
